src/TrackingRouter.js

```diff
@@ -9,10 +10,14 @@ class TrackingRouter extends AbstractRouter {
     }
 
     async content(req: Request): Promise<any> {
-        let routeID = req.query.routeID;
-        const AuthStr = 'Bearer 5a54bc7f-a7df-3796-a83a-5bba7a8e31c8';
         try {
-        let trackingRequest = await axios.get('https://realtimetcatbus.availtec.com/InfoPoint/rest/Vehicles/GetAllVehiclesForRoute?routeID=' + routeID, {headers: {Authorization: AuthStr}});
+        let parameters: any = {
+        	routeID: req.query.routeID
+        };
+        let trackingRequest = await axios.get('https://realtimetcatbus.availtec.com/InfoPoint/rest/Vehicles/GetAllVehiclesForRoute', {
```

Comment:
This url should actually be from the official endpoint, like the AllBuStopsRouter. Change it to that and also use the accessToken